content/common/cli-reference.md

```diff
@@ -105,6 +105,37 @@ SUBCOMMANDS:
 
 {{ blockEnd }}
 
+{{ startTab "v1.2.0"}}
```

Comment:
The 1.0 tab is still in here; what is our policy on keeping docs for old versions around?

Reply:
We decided to strip out anything before v1.0.0 (we removed 0.x.0).
As far as moving forward, we might all need to weigh in on that. I don't have a preference other than we should probably only have a maximum of 5 tabs at any one time. I am even flexible on that. :)